lib/api/controllers/securityController.js

```diff
@@ -0,0 +1,61 @@
+var
+  ResponseObject = require('../core/models/responseObject');
+
+module.exports = function SecurityController (kuzzle) {
+
+  /**
+   * Get a specific role according to the given id
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getRole = function (requestObject) {
+    kuzzle.pluginsManager.trigger('data:getRole', requestObject);
+
+    return kuzzle.repositories.role.loadRole(
+      kuzzle.repositories.role.getRoleFromRequestObject(requestObject)
+      )
+      .then(role => {
+        return Promise.resolve(new ResponseObject(requestObject, role || {}));
```

Comment:
Because this is not a ES document (for the user) I'm wondering if we have to add a `_source` attribute (we don't have `_id`, `_version`, etc)
Like for getStats and other function like this, we don't have a `_source` attribute.
Do think it's better to add `_source` in any cases ? If we add it here, we have to add everywhere